internal/plugin.go

```diff
@@ -648,6 +651,41 @@ func (p *Plugin) assertValidConfig() error {
 		)
 	}
 
+	// validate placement and binding names are DNS compliant
+	if p.PolicyDefaults.Placement.PlacementRuleName != "" {
+		placementName := p.PolicyDefaults.Placement.PlacementRuleName
+		if placementName != "" && len(validation.IsDNS1123Subdomain(placementName)) > 0 {
```

Comment:
Could you combine this with the previous if statements? Same goes for the others below.

Reply:
what do you mean here? I split them into separate if statements since we didn't want to prioritize certain fields

Reply:
Oh, I meant:
```go
if p.PolicyDefaults.Placement.PlacementRuleName != "" && len(validation.IsDNS1123Subdomain(placementName)) > 0 {
```